bundle/manifests/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -308,14 +312,20 @@ spec:
                 - --secure-listen-address=0.0.0.0:8443
                 - --upstream=http://127.0.0.1:8080/
                 - --logtostderr=true
-                - --v=10
-                image: registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:f000432f07cd187469f0310e3ed9dcf9a5db2be14b8bab9c5293dd1ee8518176
+                - --v=0
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.11.0
```

Comment:
Doesn't this still need to be the `ose-kube-rbac-proxy` right?

Reply:
My stance on this:

This should be considered "upstream". Which means, it should "just work". If a brand new operator were to be scaffolded by operator-sdk, this is what you would get. The change to ose-kube-rbac-proxy would happen at the pinning step.

Reply:
👍

Reply:
@bcrochet What pinning step are you referring to?